js/views/PageNav.js

```diff
@@ -225,10 +225,18 @@ export default class extends BaseVw {
   }
 
   navMaxClick() {
-    if (remote.getCurrentWindow().isMaximized()) {
-      remote.getCurrentWindow().unmaximize();
+    if (remote.process.platform === 'darwin') {
```

Comment:
There's no need to do the check for the platform, Windows also supports fullScreen, and that's the expected behavior when clicking this button. 

It's also what the right click, view, toggle fullscreen option does.

Using maximize originally was probably just an oversite.

Take out the if and just do the part with setFullscreen.

If you want to be very concise, you can do it as:
```
navMaxClick() {
    remote.getCurrentWindow().setFullScreen(!remote.getCurrentWindow().isFullScreen());
  }
```

Reply:
So we understand... this type of behavior of maximize is for new OSs. I remember when I used Windows, maximize didn't do full screen. I'm on OS mavericks and it doesn't do fullscreen for maximize either.

I have no problem with this change, but we'll probably get a bug or two from users on old OSs, who may not like and I think we'd have to live with that. I certainly wouldn't want to have to detect the OS version and selectively decide based off of that.

Reply:
> There's no need to do the check for the platform, Windows also supports fullScreen, and that's the expected behavior when clicking this button.

What about Linux? Linux may do a more traditional maximize (i.e. not fullscreen).